docker-compose.yml

```
@@ -1,59 +1,143 @@
+      - 9735  # P2P
+      - 10001 # RPC
+      - 8081  # REST
     ports:
-      - "10001:10001" 
-      - "9735:9735"
-      - "8081:8081"
+      - '${MOSTRO_LND_RPC_LOCAL_PORT:-10001}:10001'
+      - '${MOSTRO_LND_REST_LOCAL_PORT:-8081}:8081'
+    volumes:
+      - mostro_lnd:/home/lnd/.lnd
+  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)